guides/GuideForGuests.md

```diff
@@ -0,0 +1,88 @@
+# Guide for Guests
+
+Firstly, THANK YOU so much for agreeing to appear on our Open Science demo calls!!
+
+We are so so excited to highlight all of the amazing work that's happening in the OHBM Open Science community, but we couldn't do it without you giving up your time to join the calls. Thank you.
+
+![](https://media.giphy.com/media/11sBLVxNs7v6WA/giphy.gif)
+
+This guide should talk you through what you need to know to prepare for the call. But you can always send an email to Kirstie Whitaker at [kw401@cam.ac.uk](mailto:kw401@cam.ac.uk) if you need any clarification.
+
+---
+
+***tl;dr: join the google hangout 5 mins early, complete [this google form](https://goo.gl/forms/3foIuNP9G7kmCW193) in advance & have fun*** :sparkles:
+
+---
+### What to expect
+
+The Open Science demo call will start at **7pm GMT** and finish at 7:30pm GMT. We're aiming for a short but sweet taster of your project and aim to "always leave them wanting more" :wink:
+
+There will only be four of us on the call: your host [Kirstie Whitaker](https://github.com/KirstieJane), two other guests, and yourself. We'll connect via [Google Hangouts](https://hangouts.google.com/) so please make sure you have a google account and that you're using a browser compatible with hangouts (at time of writing, [Firefox does not support Google Hangouts](https://gsuiteupdates.googleblog.com/2017/02/google-hangouts-temporary-issues-with-firefox.html?m=1) but Chrome (unsurprisingly) does).
+
+The call will be [live streamed on YouTube](https://www.youtube.com/channel/UChvSitFvqGDeA1y7MJs4CGQ). People watching will be able to ask questions at the OHBM Open Science channel and we can answer them either verbally during the call or as replies via the live chat. In an ideal situation we'll have more questions than we can address in the call, so please feel free to answer questions about your project while the other guests are being interviewed.
+
+During the call Kirstie will ask you a few questions about your project. They're described in more detail below, but the most important point is to stay high level, personal and fun. We'd like these demo calls to show off the **people** behind the projects, and to tell some of the stories that are difficult to learn from git commit messages alone.
```

Comment:
This ("we'd like the demo calls to show off the **people** behind the projects, and to tell....") could be confusing to a guest who is the sole developer on their project, since it seems to imply a need for more than one person.

Could we instead say "we'd like the demo calls to focus on you (and your team) as the **people** behind the projects, and to tell...."?

Reply:
Done.